src/stream/src/cache/lru_manager.rs

```diff
@@ -91,18 +92,27 @@ impl LruManager {
 
         let mut watermark_time_ms = Epoch::physical_now();
         let mut last_total_bytes_used = 0;
+        let mut cur_total_bytes_used;
         let mut step = 0;
 
+        let jemalloc_epoch_mib = jemalloc_epoch::mib().unwrap();
+        let jemalloc_allocated_mib = jemalloc_stats::allocated::mib().unwrap();
+
         let mut tick_interval =
             tokio::time::interval(Duration::from_millis(self.barrier_interval_ms as u64));
+
         loop {
             // Wait for a while to check if need eviction.
             tick_interval.tick().await;
 
-            let stats = INSTRUMENTED_JEMALLOC.stats();
-            let cur_total_bytes_used = stats
-                .bytes_allocated
-                .saturating_sub(stats.bytes_deallocated);
+            if let Err(e) = jemalloc_epoch_mib.advance() {
+                tracing::warn!("Jemalloc epoch advance failed! {:?}", e);
+            }
+
+            cur_total_bytes_used = jemalloc_allocated_mib.read().unwrap_or_else(|e| {
```

Comment:
Have you tested whether the value is correct? Just to make sure it works.

Reply:
Yes.